### PR TITLE
Update time_series.ipynb

### PR DESCRIPTION
**Pull request by christophmeyer**
Minor correction in the text: target_size is the size of the label not the label itself.

### CODE REVIEW

**Comment by MarkDaoust** [comment]
Or maybe upgrade it to use [`TimeSeriesGenerator`](https://www.tensorflow.org/api_docs/python/tf/keras/preprocessing/sequence/TimeseriesGenerator)

**Comment by lamberta** [comment]
PR stalled and out of sync. Thanks, though
